tests/integration/test_auth_proxy.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Integration tests for jenkins-k8s-operator with auth_proxy."""
+
+# pylint: disable=unused-argument
```

Comment:
I think it might be possible to use a decorator to avoid needing this disable, see https://jdkandersson.com/2023/08/18/how-to-avoid-pylint-disableunused-argument-in-pytest/

Reply:
Nice! Thanks for the tip. Updated